### DISCUSSION TITLE
How to disable/disallow responses in a trial with the canvas-slider-response plugin?

### DISCUSSION THREAD

**Discussion by prashantig25** [jspsych/jsPsych#2372] [Q&A] [answered]
Dear All,

I have a trial in my experiment where participants are presented with a canvas drawn object and a slider is presented to the participants. However, I don't want participants to respond to this using the slider. That is, I want the participants to just see the prompt, slider, and the canvas-drawn object. Is there a way to disallow responses? Or a way to disable the slider, so that they can't respond?

Thank you,
Prashanti

**Comment by becky-gilbert** [comment, marked as answer]
Hi @prashantig25,
I don't think there's a way to do this using the plugin's existing parameters. One option is to use the trial's `on_load` function to access the slider element and disable it. I haven't tried this but I think you could do something like this:
```js
var slider_trial = {
  // ...
  on_load: function () {
    var slider = document.getElementsByTagName('input')[0];
    slider.disabled = true;
  }
}
```
Does that work?

**Reply by prashantig25** [reply]
That does work! Thank you so much.
